### PR TITLE
fix data returned on signin

### PR DESCRIPTION
**Pull request by EngrEric**

### CODE REVIEW

**Comment by enyeInc** [comment]
@EngrEric Looks good to me, just need a version bump

**Comment by Simpcyclassy** [comment]
Looks good to me

**Comment by joshuaobinna** [comment]
Looks good to me

File: routes/signInAndOut/helpers.js [hunk 1]
```diff
@@ -69,10 +70,11 @@ export default {
 
             const userSession = Object.assign(new UserSession(), { userId: user._id  });
             const doc = await userSession.save();
-
+            const userInfo = _pick(user, USERINFO);
+            
             await bcrypt.compare(password, user.password, (error, result) => {
                 const data = result ?
-                    { message: SUCCESSFUL_SIGNIN, success: true, token: doc._id } :
+                    { message: SUCCESSFUL_SIGNIN, success: true, token: doc._id, userInfo } :
```
[the remark this answers is not in the document]
Comment (EngrEric) [reply routes/signInAndOut/helpers.js]: cool, i think this is good to also be added at engineering conventions repo

**Comment by joshuaobinna** [comment]
looks good to me

File: routes/signInAndOut/helpers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Way2nnadi) [reply routes/signInAndOut/helpers.js]: yeah I will do that right now

**Comment by Way2nnadi** [comment]
@EngrEric This is ready to be merged down

**Pull request merged**